src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -10,17 +10,18 @@
 		<AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
 		<PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
 		<PackageReleaseNotes>	<![CDATA[
-        - Added table name in dataset
-		- Added get Dataplane access API
```

Comment:
Just to be sure, are you planning to bump the version in a different PR

Reply:
The version is already bumped by Meiyan in her PR to 3.0.0. That has not been released yet.